fpga/scripts/bind_platform_vfio.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Bind ips to vfio#
+# Pass the device tree names e.g. "a0000000.dma a291229.switch ..." as comandline arguments
+#
+# Author: Pascal Bauer <pascal.bauer@rwth-aachen.de>
+# SPDX-FileCopyrightText: 2023-2024 Pascal Bauer <pascal.bauer@rwth-aachen.de>
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
i'm not sure if this is the correct SPDX format. I think the copyright needs to come first?